README.md

```diff
@@ -59,7 +59,62 @@ ODK Central Frontend communicates with ODK Central Backend in part using a sessi
 
 - To upload an XLSForm, run [pyxform-http](https://github.com/getodk/pyxform-http). ODK Central Frontend communicates with pyxform-http through ODK Central Backend.
 - You can use ODK Collect to scan an app user QR code, download a form, and submit data. One option to do so is to use [`ngrok`](https://ngrok.com/download). ODK Central Frontend is available on port 8989, so you can run `ngrok http 8989` to expose a temporary HTTPS URL that you can use. Within ODK Central Backend, you will also need to set the `default.env.domain` property in [`config/default.json`](https://github.com/getodk/central-backend/blob/master/config/default.json) to the HTTPS URL, then restart ODK Central Backend if it is already running.
-- Enketo is a web form engine used to show form previews and allow for web-based data entry. The development `nginx.conf` is not configured to work with Enketo.
+- Enketo is a web form engine used to show form previews and allow for web-based data entry. The following section describes how to set up an Enketo server for use by ODK Central Frontend and ODK Central Backend.
+
+## (Optional) Running Enketo for development
+
+Enketo provides a web-based form submission UI, which is used in ODK Central for previewing forms, collecting new Submissions through Public Access Links, and editing Submissions. The following configuration is only required if you need this functionality in development. Enketo is automatically installed as part of the production ODK Central stack (via the [`central`](https://github.com/getodk/central) repository).
+
+Enketo runs as a Node.js server and caches intermediate representations of forms in Redis. ODK Central Backend stores information for managing and launching Enketo forms from ODK Central Frontend, so all three must be configured together.
+
+- Clone [enketo-express](https://github.com/enketo/enketo-express) and read the [Manual Installation Instructions](https://github.com/enketo/enketo-express). Specifically:
+	* Install Redis. (Node.js will already be installed to run ODK Central Frontend and Backend.) Enketo will expect Redis on the default port of `6379`.
+	* Install dependencies with `npm install`.
+- Configure Enketo. A minimal `config/config.json` for Enketo looks like:
+
+```
+{
+    "port": "8005",
+    "linked form and data server": {
+        "server url": "localhost:8989",
+        "api key": "enketorules",
+        "authentication": {
+            "allow insecure transport": "true"
+        }
+    },
+    "base path": "-",
+    "redis": {
+        "cache": {
+            "port": "6379"
+        }
+    }
+}
+```
+
+- Configure ODK Central Backend. Add the following to `config/default.json`.
+
+```
+    "enketo": {
+      "url": "http://localhost:8005/-",
+      "apiKey": "enketorules"
+    },
```

Comment:
Similar to the above, I'm also wondering whether we should go ahead and put this in `config/default.json` now (though I feel less sure about that). Currently, `config/default.json` doesn't specify a configuration for Enketo, but it does specify a configuration for pyxform-http (localhost:5000). If pyxform-http isn't running, then the developer will see an error message, but if it is, the developer doesn't need to do anything extra to get Backend to start communicating with pyxform-http: it works automatically. Similarly, I think that if we add this configuration to `config/default.json`, then if Enketo isn't running, the developer will see an error message, but if it is, things will start working right away. I think right now, the developer will also see an error message, so I'm thinking there might not be much downside to making that change.

Reply:
I think that's right but would be good to review how/if `config/default.json` is used to get to a prod setup. I'd merge this first and maybe explore next time we have to look at Enketo config things.